tests/e2e/e2e_smi_traffic_target_test.go

```diff
@@ -48,6 +48,11 @@ var _ = OSMDescribe("Test HTTP traffic with SMI TrafficTarget",
 					})
 				_, err := Td.CreateServiceAccount(destName, &svcAccDef)
 				Expect(err).NotTo(HaveOccurred())
+				if Td.DeployOnOpenShift {
```

Comment:
you probably already thought of this, we can move this code on the testers themeselves (`Td.CreateServiceAccount`).

If the helpers become openshift-aware, it will probably allow the entire test-suite to run on openshift

Reply:
That's a great idea Edu. The only question I have is that this would require the user to pass in the SCC to the CreateServiceAccount function because this needs to be configurable (for apps it needs to be "privileged" for other things it needs to be "hostaccess"). If we're okay with CreateServiceAccount to have an SCC argument, I can go ahead with the change.

Reply:
if we add an additional param on that function with a comment saying that is only used when in `deployOnOpenShift` and it's already enough IMO. Do you suspect a lot more changes config-wise for openshift?

Reply:
changed to hardcode privileged SCC for now